generate-task-list.js

```diff
@@ -0,0 +1,59 @@
+function toCamelCase(text) {
+	return text.replace(/^([A-Z])|[\s-_]+(\w)/g, function (match, p1, p2, offset) {
+		if (p2) return p2.toUpperCase();
+		return p1.toLowerCase();
+	});
+}
+
+function camelCaseToString(str, separator) {
+	separator = typeof separator === 'undefined' ? '_' : separator;
+
+	return str
+		.replace(/([a-z\d])([A-Z])/g, '$1' + separator + '$2')
+		.replace(/([A-Z]+)([A-Z][a-z\d]+)/g, '$1' + separator + '$2')
+		.toLowerCase();
+}
+
+const addTaskForm = document.querySelector('.task-form__add');
+const taskList = document.querySelector('.task-list');
+const addTask = event => {
+	event.preventDefault();
+
+	const taskName = document.querySelector('task-form__input', 'task-form__input--task-content').value;
+	const taskDeadline = document.querySelector('task-form__input', 'task-form__input--date').value;
+
+	let lastElement = tasks.pop();
+
+	tasks.push({
+		id: lastElement.id + 1,
+		name: toCamelCase(taskName),
```

Comment:
Actually there's no need to store task names in camel case, because when you render the name in some task list, you need to de-camelcase it again. Or maybe I'm missing something?

Reply:
So tasks array should be like : 
`let tasks = [{
		id: 1,
		name: 'Go to cinema',
		deadline: '12.04.2019',
		isDone: false,
		isDeleted: false
	},`

or 

`let tasks = [{
		id: 1,
		name: 'goToCinema',
		deadline: '12.04.2019',
		isDone: false,
		isDeleted: false
	},`

Reply:
The former